src/wp-includes/functions.php

```diff
@@ -8371,3 +8371,17 @@ function is_php_version_compatible( $required ) {
 function wp_fuzzy_number_match( $expected, $actual, $precision = 1 ) {
 	return abs( (float) $expected - (float) $actual ) <= $precision;
 }
+
```

Comment:
I think this function can be moved to `wp-includes/l10n.php`, for consistency with other locale-specific functions.